### ISSUE TITLE
exp_lr_scheduler is missing

### ISSUE THREAD

**Issue by gudovskiy** [amirgholami/HessianFlow#1]
Seems that you need this to utils.py:

def exp_lr_scheduler(optimizer, decay_ratio):
    """Decay learning rate by a factor of lr_decay every lr_decay_epoch epochs"""    
    for param_group in optimizer.param_groups:
        param_group['lr'] *= decay_ratio
    return optimizer

**Comment by gudovskiy** [comment]
OK, I found this function in ./optimizer . Then, call should be:
hf_optm_utils.exp_lr_scheduler(optimizer, decay_ratio=args.lr_decay)
